.github/workflows/faucet.yml

```
@@ -66,4 +66,4 @@ jobs:
     uses: ./.github/workflows/release-docker-images.yml
     with:
       app_name: faucet
-    secrets: inherit
+    secrets: inherit
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)